perf/metrics.go

```diff
@@ -0,0 +1,59 @@
+package perf
+
+import (
+	"fmt"
+	"log"
+
+	"go.opencensus.io/stats/view"
+	"go.uber.org/multierr"
+)
+
+// RegisterApplicationViews registers all the default views we may need for the application. gRPC, MongoDb, http, etc...
+func registerApplicationViews() error {
+	return multierr.Combine(
+		registerGrpcViews(),
+		registerHTTPViews(),
+	)
+}
+
+// indent these many spaces.
+const indent = "  "
+
+// NewPrintExporter creates a new metric exporter that prints to the console.
+// This should NOT be used for production workloads.
+func NewPrintExporter() view.Exporter {
+	return &PrintExporter{}
+}
+
+// PrintExporter is a stats and trace exporter that logs
+// the exported data to the console.
+//
+// The intent is help new users familiarize themselves with the
+// capabilities of opencensus.
+//
+// This should NOT be used for production workloads.
+type PrintExporter struct{}
+
+// ExportView logs the view data.
+func (e *PrintExporter) ExportView(vd *view.Data) {
```

Comment:
can you add somewhere in a readme or doc.go how to use all this stuff? either via GCP / a local server/viewer/printer?

Reply:
I added a few examples to exporters_test.go